### DISCUSSION TITLE
Mark draft PR as ready for review

### DISCUSSION THREAD

**Discussion by legrego** [community/community#3593] [Mobile] [answered]
Perhaps I missed it, but I'd love the ability to mark a draft PR as ready for review from the mobile app.

**Comment by candyho** [comment, marked as answer]
Hey thanks for your feedback. I've tracked this request in our internal repo. Hopefully we'll have an update to share with you soon.

**Comment by viceice** [comment]
This is working now on Android app, but I need the opposite. So i like to mark a PR as draft.

**Comment by dan-r** [comment]
Still not seeing this in the iOS app - have I missed something?
